internal/proxy/task_delete.go

```diff
@@ -169,12 +169,12 @@ func (dt *deleteTask) PreExecute(ctx context.Context) error {
 
 	collName := dt.CollectionName
 	if err := validateCollectionName(collName); err != nil {
-		log.Error("Invalid collection name", zap.String("collectionName", collName))
+		log.Error("Invalid collection name", zap.String("collectionName", collName), zap.Error(err))
 		return err
 	}
 	collID, err := globalMetaCache.GetCollectionID(ctx, collName)
 	if err != nil {
-		log.Debug("Failed to get collection id", zap.String("collectionName", collName))
+		log.Error("Failed to get collection id", zap.String("collectionName", collName), zap.Error(err))
```

Comment:
It's good to keep it as Debug, which is consistent with the  case of cache miss for partition.